staging/src/k8s.io/kube-openapi/pkg/aggregator/aggregator.go

```diff
@@ -474,9 +514,15 @@ func MergeSpecs(dest, source *spec.Swagger) error {
 
 // Clone OpenAPI spec
 func CloneSpec(source *spec.Swagger) (*spec.Swagger, error) {
-	if ret, err := cloner.DeepCopy(source); err != nil {
+	// TODO(mehdy): Find a faster way to clone an spec
```

Comment:
how often do we clone a spec? Once per APIService change? That's probably not often.